android/anc/src/main/java/org/smartregister/fhircore/anc/ui/anccare/details/AncDetailsFragment.kt

```diff
@@ -155,20 +151,28 @@ class AncDetailsFragment : Fragment() {
         binding.txtViewNoCarePlan.visibility = View.VISIBLE
         binding.txtViewCarePlanSeeAllHeading.visibility = View.GONE
         binding.imageViewSeeAllArrow.visibility = View.GONE
-        binding.carePlanListView.visibility = View.GONE
+        binding.txtViewCarePlan.visibility = View.GONE
       }
       else -> {
         binding.txtViewNoCarePlan.visibility = View.GONE
         binding.txtViewCarePlanSeeAllHeading.visibility = View.VISIBLE
         binding.imageViewSeeAllArrow.visibility = View.VISIBLE
-        binding.carePlanListView.visibility = View.VISIBLE
+        binding.txtViewCarePlan.visibility = View.VISIBLE
```

Comment:
ENHANCEMENT (for readability): There are two extension methods for the `View` class that can be used to `show` or `hide` a view.

```
        binding.apply {
              txtViewNoCarePlan.hide()
              txtViewCarePlanSeeAllHeading.show()
              imageViewSeeAllArrow.show()
              txtViewCarePlan.show()
         }
````